src/sagas/server-configs.js

```diff
@@ -60,6 +62,14 @@ function* fetchUSBFilter () {
   }
 }
 
+function* fetchSpiceUsbAutoShare () {
+  const autoShare = yield callExternalAction('getSpiceUsbAutoShare', Api.getSpiceUsbAutoShare, {})
```

Comment:
```suggestion
  const autoShare = yield callGetOption('getSpiceUsbAutoShare', 'general', 1)
```

Follow the pattern for fetching the other config values.  The generic `callGetOption()` lets us skip having to add a new API function for each setting.